shared/chat/conversation/messages/message-popup/exploding/container.js

```diff
@@ -97,6 +108,13 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
       title: 'Explode now',
     })
   }
+  if (isMobile) {
```

Comment:
same here, just check dispatchProps._onAddReaection and have the isMobile check there. I think i prefer these menuitem things to be made in the component and not in the connector but we can leave this for now

Reply:
the attachment popup does the check similarly in `mergeProps`, so it seems kinda arbitrary to move it up a level here. I'm changing the other ones over though.

Reply:
I also see why making the menuitems in the component would be nice for storybook and such. on the other hand, I went with the container here because using undefined props to switch the view is pretty difficult to read IMO. 🤔